code/game/machinery/washing_machine.dm

```diff
@@ -44,15 +45,27 @@
 	for(var/obj/item/clothing/suit/hooded/ian_costume/IC in contents)
 		new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/corgi(src)
 		qdel(IC)
-
+	
+	for(var/obj/item/weapon/paper/P in contents)
+		if(crayon)
+			var/dye_color
+			if(istype(crayon,/obj/item/toy/crayon))
+				var/obj/item/toy/crayon/CR = crayon
+				dye_color = CR.paint_color
+			if(dye_color)
+				P.color = dye_color
+			qdel(crayon)
+			crayon = null

```

Comment:
yeah, the whole PR is a mess, 90+ commits. And robustin did some weird branch juggling, looking at this change in the history of washing_machine.dm doesn't lead you to this PR but to a Pull Request on robustin's tg branch. I hate when that happens cause it makes reaching the source PR a pain in the ass.